bears/general/URLBear.py

```diff
@@ -18,6 +20,28 @@ class LINK_CONTEXT(Flag):
     pip_vcs_url = 2
 
 
+class URLResult(HiddenResult):
```

Comment:
you need to add `generate_ordering` and `generate_repr` decorators as the inherited methods are wrong.
c.f. https://github.com/coala/coala/issues/4457

A new test for the `repr(URLResult)` result would be good.
I wouldnt worry about the test for the ordering methods.